roles/lnls-ans-role-sirius-hla/templates/sirius_hla.sh.j2

```diff
@@ -0,0 +1,12 @@
+# EDM
+# ===
+
+export EDMPVOBJECTS=/etc/edm
+export EDMOBJECTS=/etc/edm
+export EDMHELPFILES=/usr/share/edm/helpFiles
+SIRIUS_APPS_EDMDATAFILES=/usr/share/edm/data:/usr/local/share/edm/data
```

Comment:
Wrong variable name, should be `HLA_APPS`

Reply:
changed local variable